esp/src/src/Memory.ts

```diff
@@ -92,7 +92,9 @@ export class Memory extends BaseStore {
         }
     }
 
+    dataVersion = 0;
     setData(data) {
+        this.dataVersion++;
```

Comment:
What was this used for?

Reply:
Its used to trigger a re-render in React